packages/dds/sequence/src/test/intervalCollection.snapshot.spec.ts

```diff
@@ -88,6 +120,15 @@ describe("IntervalCollection snapshotting", () => {
         containerRuntimeFactory.sequenceNumber = seq;
     });
 
+    // **** don't exactly know what to do with this? ****
+    it("loads the old interval format", async () => {
```

Comment:
I'm not sure you particularly need changes to this test file. What was the intent here?

Reply:
Left this in on accident, reverted